src/components/invite/useInvites.js

```diff
@@ -167,26 +174,55 @@ const useInvited = () => {
 
       log.debug('updateInvited got invitees profiles', { invited })
 
-      //keep if both name + avatar or not in cache
-      invited = invited.filter(_ => _.name || _.avatar || cached.find(c => c.addr === _.addr) === undefined)
+      // keep if both name + avatar or not in cache
```

Comment:
i rewrote this and the following `.forEach` because it's really hard to understand those JAPH-like expressions